client/lib/discounts/index.js

```diff
@@ -5,12 +5,14 @@
  */
 import activeDiscounts from './active-discounts';
 
-function getDiscountByName( discountName ) {
+function getDiscountByName( discountName, endsAt ) {
```

Comment:
_Maybe_ add a default value in case it gets called with this argument?
eg. `endsAt = null`
Probably fine without that too.